coalib/bears/Bear.py

```diff
@@ -72,6 +81,13 @@ def supported_languages(cls):
         return (cls.LANGUAGES if isinstance(
             cls.LANGUAGES, tuple) else (cls.LANGUAGES,))
 
+    @classproperty
+    def requirements(cls):
+        """
+        :return: The requirements needed by the bear.
+        """
+        return cls.REQUIREMENTS
```

Comment:
hm I think this property is actually not needed, you have a field which you can access like the property, imo this is redundant :+1: